configuration/configurator/ConfiguratorMain.cpp

```diff
@@ -197,13 +195,13 @@ void usage()
             }
             if (argv[idx][0] != '.')
             {
-                strcat(pTargetDocExt, ".");
-                strcpy(&(pTargetDocExt[1]),argv[idx]);
+                pTargetDocExt = ".";
             }
-            else if (*pTargetDocExt == 0)
-                strcat(pTargetDocExt, "");
             else
-                strcpy(pTargetDocExt,argv[idx]);
+            {
+                pTargetDocExt = "";
```

Comment:
Changed to use explicit method instead of overloaded operator.

@richardkchapman Curious your take on use of overloaded operators versus explicit method calls. Thanks.